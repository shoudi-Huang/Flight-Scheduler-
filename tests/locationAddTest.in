LocATioN aDd
location add Mumbai 19.08 72.74
location add Mumbai kk 72.741 0.4
location add Mumbai -90 72.741 0.4
location add Mumbai 86 72.741 0.4
location add Mumbai 19.08 lon 0.4
location add Mumbai 19.08 -190 0.4
location add Mumbai 19.08 181 0.4
location add Mumbai 19.08 72.741 qd
location add Mumbai 19.08 72.741 1.2
location add Mumbai 19.08 72.741 -2
location add Mumbai 19.08 72.741 0.4
location add Mumbai 19.08 72.741 0.4
location add NewDelhi 28.527 77.0688988 -0.123 k l o 9
location Mumbai
location NewDelhi
exit
