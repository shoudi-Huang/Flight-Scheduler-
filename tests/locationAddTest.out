User: Usage:   LOCATION ADD <name> <lat> <long> <demand_coefficient>
Example: LOCATION ADD Sydney -33.847927 150.651786 0.2

User: Usage:   LOCATION ADD <name> <lat> <long> <demand_coefficient>
Example: LOCATION ADD Sydney -33.847927 150.651786 0.2

User: Invalid latitude. It must be a number of degrees between -85 and +85.

User: Invalid latitude. It must be a number of degrees between -85 and +85.

User: Invalid latitude. It must be a number of degrees between -85 and +85.

User: Invalid longitude. It must be a number of degrees between -180 and +180.

User: Invalid longitude. It must be a number of degrees between -180 and +180.

User: Invalid longitude. It must be a number of degrees between -180 and +180.

User: Invalid demand coefficient. It must be a number between -1 and +1.

User: Invalid demand coefficient. It must be a number between -1 and +1.

User: Invalid demand coefficient. It must be a number between -1 and +1.

User: Successfully added location Mumbai.

User: This location already exists.

User: Successfully added location NewDelhi.

User: Location:    Mumbai
Latitude:    19.080000
Longitude:   72.741000
Demand:      +0.4000

User: Location:    NewDelhi
Latitude:    28.527000
Longitude:   77.068899
Demand:      -0.1230

User: Application closed.
